axum/src/body/mod.rs

```diff
@@ -8,3 +8,17 @@ pub use bytes::Bytes;
 
 #[doc(inline)]
 pub use axum_core::body::Body;
+
+use http_body_util::BodyExt;
+
+/// Converts [`Body`] into [`Bytes`].
+/// ```rust
+/// # use axum::body::{to_bytes, Body};
+/// async fn foo() {
+///     let body = Body::from(vec![1, 2, 3]);
+///     assert_eq!(&to_bytes(body).await.unwrap()[..], &[1, 2, 3]);
+/// }
+/// ```
```

Comment:
```suggestion
/// Converts [`Body`] into [`Bytes`].
///
/// # Example
///
/// ```rust
/// use axum::body::{to_bytes, Body};
///
/// # async fn foo() -> Result<(), axum::Error> {
/// let body = Body::from(vec![1, 2, 3]);
/// let bytes = to_bytes(body).await?
/// assert_eq!(&bytes[..], &[1, 2, 3]);
/// # Ok(())
/// # }
/// ```
```

Reply:
Fixed